src/client/routes/Labs/index.tsx

```diff
@@ -481,8 +483,7 @@ export const Labs = () => {
                 </SearchBarContainer>
               }
               searching={opportunities.length > filteredOpportunities.length}
-              // TODO Redirect address is wrong
-              // onAction={({ address }) => history.push(`/vault/${address}`)}
+              onAction={({ address }) => history.push(`/lab/${address}`)}
```

Comment:
Also need same fix [here](https://github.com/yearn/yearn-finance-v3/pull/561/files#diff-656bc72c3ac9fd94e17e141c522cd0a9c54a12dae5221f8b981f091dc44a7d65R411)

Reply:
What fix? I'm confused :)

Reply:
You missed adding the `history.push` fix here https://github.com/yearn/yearn-finance-v3/blob/54d4240ba6ebb586850fd9bb1f9e79b5864e665d/src/client/routes/Labs/index.tsx#L411

Reply:
Ahhh fixed with https://github.com/yearn/yearn-finance-v3/pull/561/commits/ec31c34f3dc2028038661cf909f60e47058983d1